### ISSUE TITLE
[Bug] Propagation fails when adding new sites

### ISSUE THREAD

**Issue by OwenMelbz** [verbb/navigation#260]
**Description**
When you add a new site to your installation with a pre-existing populated navigation.

You then "enable" the navigation on that site and tick the "Propagate" option.

It doesn't actually propagate them across, it leaves the other sites completely blank.

We've had 3 different websites with this issue now so pretty confident that it is related to the Plugin and not a freak accident.

**Additional info**
- Plugin version: Latest
- Craft version: Latest
- Multi-site: Yes or No
- Propagate nodes: Yes or No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.17",
  "...": "..."
}
```

Then run `composer update`. Do note that it requires toggling off propagation, then back on, but it'll work for existing sites, so you don't have to re-create the site entirely.

**Comment by OwenMelbz** [comment]
Hi,

Thanks for looking into this - Do you have any estimations as to when the next release will be made?

Thanks

**Comment by OwenMelbz** [comment]
@engram-design Hi, just an FYI I think this only 1/2 worked.

We use nesting using a Passive link e.g.

Before:

![image](https://user-images.githubusercontent.com/1094740/121868500-fb990f80-ccf8-11eb-88d8-5f14fbb447a2.png)

However after we enabled propagation it flattened everything and changes the orders e.g.

After:

![image](https://user-images.githubusercontent.com/1094740/121868583-14a1c080-ccf9-11eb-94e2-9f57acb24534.png)

**Comment by OwenMelbz** [comment]
Additionally things are starting to implode :D the more sites we add it seems to clone the entries over and over

![image](https://user-images.githubusercontent.com/1094740/121904593-3adb5680-cd21-11eb-88ba-cabd180f9cca.png)

![image](https://user-images.githubusercontent.com/1094740/121904747-5fcfc980-cd21-11eb-8f19-cbc5836eff8c.png)

**Comment by engram-design** [comment]
Thanks for following up, indeed there were still some issues, which I've ironed out. You might want to clear the nodes for each site to start again (sorry).

Should be fixed in [1.4.18](https://github.com/verbb/navigation/releases/tag/1.4.18)

The second instance here is unfortunately going to happen if you keep toggling off/on the propagation settings for a nav as well. The reason is that with propagation turned on, all nodes across sites use the same element ID for each node (just changing the siteId for the appropriate site). However, when you turn off propagation, you're essentially saying "unlink" each node from each other, and be independent. This means we need to create new node elements for each site, which makes it impossible to match again if toggling back on, so you'll be bound to end up with duplicates. The only sure-fire way to ensure this doesn't happen is to wipe all non-primary navigations, and copy from the primary site navigation fresh.

The only real issue you might face is if you turn off "Propagate nodes across all enabled sites?", then back on - which is something I need to address, to consolidate independent nodes to be "synced" again. The only reason this is a challenge is due to data loss of previously independent nodes.

But - it shouldn't happen if you're adding new sites (or enabling/disabling them in nav settings).

Long explanation, but hopefully that should be all fixed now.
